openedx/jobs/openedxReleaseJob.groovy

```diff
@@ -0,0 +1,42 @@
+package openedx
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+/* stdout logger */
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
```

Comment:
^ setting up the stdout logger was pretty much only used when debugging issues with parsing the secret files during the seeding process, which if they were configured incorrectly, would cause the seeding fail without explanation. Seeing that you have no secret files, you can delete this. Also, the majority of the logic is in the Jenkinsfile itself, so you can get rid of the yaml import as well